modules/cluster-bare-metal-operator.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// *  operators/operator-reference.adoc
+// *  post_installation_configuration/cluster-capabilities.adoc
+
+// operators/operator-reference.adoc
+ifeval::["{context}" == "cluster-operators-ref"]
+:operator-ref:
+endif::[]
+
+// post_installation_configuration/cluster-capabilities.adoc
+ifeval::["{context}" == "cluster-capabilities"]
+:cluster-caps:
+endif::[]
+
+:_content-type: REFERENCE
+[id="cluster-bare-metal-operator_{context}"]
+ifdef::operator-ref[= Cluster Baremetal Operator]
+ifdef::cluster-caps[= Bare-metal capability]
+
+ifdef::operator-ref[]
+
+[NOTE]
+====
+The Cluster Baremetal Operator is an optional cluster capability that can be disabled by cluster administrators during installation. For more information about optional cluster capabilities, see "Cluster capabilities" in _Post-installation configuration_.
+====
+
+endif::operator-ref[]
+
+[discrete]
+== Purpose
+
+ifdef::cluster-caps[]
+
+The Cluster Baremetal Operator provides the features for the `baremetal` capability.
+
+endif::cluster-caps[]
+
+The Cluster Baremetal Operator (CBO) deploys all the components necessary to take an unprovisioned server to a fully functioning worker node ready to run {product-title} compute nodes. The CBO ensures that the metal3 deployment consisting of the Bare Metal Operator (BMO) and Ironic containers runs on one of the control plane nodes within the {product-title} cluster. The CBO also listens for {product-title} updates to resources that it watches and takes appropriate action. 
+
+ifdef::cluster-caps[]
+The bare-metal capability is required for installer provisioned installation (IPI) deployments. Disabling the bare-metal capability can result in unexpected problems with IPI deployments.
+
+[IMPORTANT]
+====
+If the bare-metal capability is disabled, users cannot interact with bare-metal clusters. Only disable the capability if there are no bare-metal nodes in your deployment.
```

Comment:
nit: "users cannot interact with bare-metal clusters" could maybe be rephrased like "the cluster cannot provision or manage bare-metal nodes" or some such to reduce the scoping?  If a user magically added a bunch of manually-provisioned nodes, they'd be able to interact with the cluster.  And it's not clear to me if that sort of cluster would be counted in the "bare-metal clusters" bucket or not.

Reply:
Thank you. I think this reads much better with your suggestion.